airflow/jobs/scheduler_job_runner.py

```diff
@@ -263,11 +263,21 @@ def __get_concurrency_maps(self, states: Iterable[TaskInstanceState], session: S
         :param states: List of states to query for
         :return: Concurrency map
         """
-        ti_concurrency_query: Result = session.execute(
-            select(TI.task_id, TI.run_id, TI.dag_id, func.count("*"))
-            .where(TI.state.in_(states))
+        ti_concurrency_query: Result = (
```

Comment:
@uranusjr is this approach fine to run single query instead of two queries ? or any other suggestion please let me know.